### ISSUE TITLE
Remake Home

### ISSUE THREAD

**Issue by bmrgould** [gctools-outilsgc/gccollab-mobile#184]
Homepage with sideways swiping content is not accessible.

User feedback also had mixed feelings about the page, but multiple complaints that it did not fit with the rest of the app. As the other content list pages all scroll down.

Replace current home with something accessible and fits with the rest of the app.
